### ISSUE TITLE
MDS milliseconds timestamp constraints

### ISSUE THREAD

**Issue by tonial** [openmobilityfoundation/mobility-data-specification#645]
### Is your feature request related to a problem? Please describe.

As an agency, we often see second timestamps instead of millisecond timestamp when a provider uses MDS for the first time and tries to send us data.

### Describe the solution you'd like

In order to speed up bug detection on our side, we added a constraint that specifies that a timestamp should have 13 digits which allows dates between 2001-9-9 and 2286-11-20 (which is big enough).
Adding this into the Timestamp description and json schema could help to prevent this mistake.

### Is this a breaking change

 * No, not breaking

### Impacted Spec

All timestamps in all APIs

**Comment by schnuerle** [comment]
Thank you @tonial. A reasonable request that could be updated in the next release. As the schema expert, what do you think @thekaveman ?

**Comment by thekaveman** [comment]
Yes, very reasonable request and should be as easy as modifying the common [timestamp definition](https://github.com/openmobilityfoundation/mobility-data-specification/blob/main/schema/templates/common.json#L378).

Right now, we have:

```json
"timestamp": {
  "$id": "#/definitions/timestamp",
  "type": "number",
  "description": "Integer milliseconds since Unix epoch",
  "multipleOf": 1.0,
  "minimum": 0
},
```

Because this is a number type, we can't do a (string) length check. But an easy modification would be to change the `minimum` to correspond to @tonial's 13-digit starting point of `1000000000000`. 

Alternatively, a more realistic minimum could correspond to (for example) Jan 1, 2018 (the year MDS was first released), in UTC `1514764800000`.

**Comment by schnuerle** [comment]
Approved via PR #648. Thank you!
